package.json

```diff
@@ -17,19 +17,17 @@
   "devDependencies": {
     "@bigtest/interactor": "0.7.2",
     "@bigtest/mirage": "^0.0.1",
-    "@folio/eslint-config-stripes": "^4.2.0",
+    "@folio/eslint-config-stripes": "^5.0.0",
     "@folio/stripes": "^2.7.4",
     "@folio/stripes-cli": "^1.11.0",
     "@testing-library/react": "^9.3.2",
     "babel-eslint": "^9.0.0",
     "babel-jest": "^23.6.0",
     "babel-plugin-transform-decorators-legacy": "^1.3.5",
     "babel-plugin-transform-runtime": "^6.23.0",
-    "eslint": "^5.6.0",
+    "eslint": "^6.2.1",
     "eslint-plugin-filenames": "^1.3.2",
     "eslint-plugin-jest": "^23.0.4",
-    "eslint-plugin-no-only-tests": "^2.3.0",
```

Comment:
we can avoid importing of the plugins if they are included in stripes eslint config